volumina/pixelpipeline/imagesources.py

```diff
@@ -52,6 +46,28 @@
 except ImportError:
     _has_vigra = False
 
+
+logger = logging.getLogger(__name__)
+
+
+def log_request(func):
+    @functools.wraps(func)
+    def _log_request(source: "ImageSource", qrect, *args, **kwargs):
+        if CONFIG.verbose_pixelpipeline:
+            logger.error(
```

Comment:
just out of curiosity, why do you use the error loglevel here?

Reply:
Initially, it was in red (and most of the output prettifiers do stderr in red) and I want to avoid setting INFO log level for the logger, because it may log something else, except pipeline requests.